### PR TITLE
Add specs

### PR DESCRIPTION
**Pull request by defnotpaul**

### CODE REVIEW

**Comment by defnotpaul** [comment]
@kainosnoema @jfhamlin - whenever y'all have a free moment, would love some feedback here

File: lib/stacker/stack/parameter.rb [hunk 1]
```diff
@@ -63,6 +63,10 @@ def to_s
         end
       end
 
+      def ==(parameter)
+        parameter.value == value && parameter.region == region
```
Comment (kainosnoema) [inline lib/stacker/stack/parameter.rb:67]: You might also check that the argument is the same class, but maybe that's too defensive.

**Review by kainosnoema — APPROVED** [review]
This is really great. Glad to finally have some tests in here. Feedback from @maraca and @jfhamlin welcome though.

File: lib/stacker/stack/parameter.rb (hunk 1 above)
Comment (defnotpaul) [reply lib/stacker/stack/parameter.rb:67]: :duck:

Comment (kainosnoema) [reply lib/stacker/stack/parameter.rb:67]: Suuuure....

**Pull request merged**